guides/micronaut-oracle-autonomous-db/micronaut-oracle-autonomous-db.adoc

```diff
@@ -261,36 +259,45 @@ You can run some cURL requests to test the application:
 
 [source,bash]
 ----
-complete $ curl -i localhost:8080/things
+curl -i localhost:8080/things
+----
+
+```
```

Comment:
I am not sure that this is proper asciidoc. I think they are Github fenced code blocks